azkaban-solo-server/src/main/resources/quartz_schema.sql

```diff
@@ -0,0 +1,249 @@
+-- Thanks to Amir Kibbar and Peter Rietzler for contributing the schema for H2 database, 
```

Comment:
I didn't see those. If either work on Postgres, should be fine on H2.

Reply:
Drop table would defeat the purpose here, since the point is to test persistent behavior and that would erase all data.

Reply:
> I didn't see those. If either work on Postgres, should be fine on H2. However, they wouldn't be available on the classpath of solo-server.

One of them, the `azkaban-web-server/src/test/resources/create.quartz-all-tables.sql`, is specifically for testing on H2. 

Speaking of which, I am beginning to think that initializing DB in AzkabanSingleServer and pointing it unconditionally to H2 may be wrong. Does Solo Server not use WebServer and ExecServer and do those two not configure and open DB according to properties? So if I have solo server configured to run against a MySQL DB, it will still create a *separate* H2 DB with separate set of Quartz tables. Does it not seem strange? Sorry, this may need more thought.

Reply:
I have implemented an additional check that only invokes the H2 script if: 1) Quartz is enabled, 2) The H2 driver is used, and 3) The table prefix is the standard one. The schema script is only executed if the database is uninitialized, as indicated by the existence of the `QRTZ_TRIGGERS` table. The schema script is specified by a configuration property, preconfigured as `azkaban-web-server/src/test/resources/create.quartz-all-tables.sql` in the solo-server config.